flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -42,9 +42,12 @@ jobmanager.rpc.port: 6123
 jobmanager.heap.size: 1024m
 
 
-# The heap size for the TaskManager JVM
+# The total process memory size for the TaskManager.
```

Comment:
```suggestion
# The total process memory size for the TaskExecutor.
```
I think this the most recent term for task manager after FLIP-6

Reply:
In the code base it is correct that we call it `TaskExecutor`. However, we never fully transitioned from `TaskManager` to `TaskExecutor` in our documentation. Given that the memory config options are prefixed with `taskmanager`, I would suggest to stick with `TaskManager` for the time being.